frog/imports/client/SingleActivity/types.js

```diff
@@ -0,0 +1,14 @@
+// @flow
+
+import { type ActivityPackageT, type ActivityDbT } from 'frog-utils';
+
+export type StateT = {
+  stage: number,
+  activityType?: ActivityPackageT,
+  activity?: ActivityDbT,
+  slug?: string
+};
+
+export type PropsT = {
+  classes: Object
```

Comment:
This is not the most appropriate solution, a better one would be to use a package like `css-modules-flow-types-loader`. Since this is not a critical issue, I'll skip this one for now.